native/cocos/platform/android/java/src/com/cocos/lib/CanvasRenderingContext2DImpl.java

```diff
@@ -416,9 +416,12 @@ private void strokeText(String text, float x, float y, float maxWidth) {
     }
 
     private void configShadow(Paint paint) {
-        if (mShadowColorA > 0 && (Math.abs(mShadowOffsetX) > Float.MIN_VALUE ||
-                                  Math.abs(mShadowOffsetY) > Float.MIN_VALUE ||
-                                  mShadowBlur > Float.MIN_VALUE)) {
+        if (mShadowColorA > 0 && (Math.abs(mShadowOffsetX) > Float.MIN_VALUE || Math.abs(mShadowOffsetY) > Float.MIN_VALUE)) {
+            if (mShadowBlur < 0) {
+                return;
+            } else if (mShadowBlur < Float.MIN_VALUE) {
+                mShadowBlur = 0.001f;//If shadowBlur is 0, the shadow effect is not consistent with the web.
```

Comment:
Here, we only make sure the radius is not equal to 0.
Android's native code:

Paint* paint = reinterpret_cast<Paint*>(paintHandle);
        if (radius <= 0) {
            paint->setLooper(nullptr);
        }
        else {
            SkScalar sigma = android::uirenderer::Blur::convertRadiusToSigma(radius);
            paint->setLooper(SkBlurDrawLooper::Make((SkColor)color, sigma, dx, dy));
        }

static const float BLUR_SIGMA_SCALE = 0.57735f;

float Blur::convertRadiusToSigma(float radius) {
    return radius > 0 ? BLUR_SIGMA_SCALE * radius + 0.5f : 0.0f;
}

Reply:
Don't need to use `else if`, just use if is enough.